projects/media-viewer/src/lib/annotations/annotation-set/annotation-set.component.ts

```diff
@@ -127,14 +127,24 @@ export class AnnotationSetComponent implements OnInit, OnDestroy {
   async createTextHighlight(highlight: Highlight) {
     if (highlight.page === this.page) {
       if (window.getSelection()) {
+        const localElement = (<HTMLElement>highlight.event.target) || (<HTMLElement>highlight.event.srcElement);
```

Comment:
Maybe: 

```
const localElement: HTMLElement = highlight.event.target || highlight.event.srcElement;
```

Reply:
this is not possible, as it doesn't covert it into HTMLElement, it just rejects it.